### DISCUSSION TITLE
Reactpy Integration in Django

### DISCUSSION THREAD

**Discussion by danieludokike** [idom-team/idom#1004] [Question] [answered]
How do I configure reactpy to work with Django?

**Comment by Archmonger** [comment, marked as answer]
You will need to use [`django-reactpy`](https://github.com/reactive-python/reactpy-django). You can check out the [`reactpy-django` documentation](https://reactive-python.github.io/reactpy-django/) for installation instructions.

**Reply by danieludokike** [reply]
Thanks so much, let me check it out

**Reply by danieludokike** [reply]
Another thing, Is there a way to tweak react router in Django and not use the Django url routing. I get minimal search result online as reactpy is a new package

**Reply by Archmonger** [reply]
We have [`reactpy-router`](https://github.com/reactive-python/reactpy-router), which is designed to be similar to `react-router`.

A deeper level of Django integration is planned for `reactpy-router`, but that is still under development.
